assignments/prototypes.js

```diff
@@ -32,79 +53,142 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid(attributes) {
+  this.team = attributes.team;
+  this.weapons = attributes.weapons;
+  this.language = attributes.language;
+  GameObject.call(this, attributes);
```

Comment:
No need for GameObject call here as inheriting from CharacterStats already does this